web/controllers/notes_controller.ex

```diff
@@ -43,4 +43,19 @@ defmodule Notox.NoteController do
        render(conn, :edit, note: note, changeset: changeset)
    end
   end
+
+  def show(conn, %{"id" => id}) do
```

Comment:
Can you change the "destination" of this to `bv-notes-show` so I don't see all the code from that branch? 

(This is one of the reason's I'd mentioned that opening multiple PRs with code that builds on top of each other isn't great)